src/Modules/ModParser.lua

```diff
@@ -2820,6 +2820,8 @@ local specialModList = {
 	["you are unaffected by bleeding while leeching"] = { mod("SelfBleedEffect", "MORE", -100, { type = "Condition", var = "Leeching" }) },
 	-- Warrior - Titan
 	["(%d+)%% increased effect of small passive skills"] = function(num) return { mod("SmallPassiveSkillEffect", "INC", num) } end,
+	-- Warrior - Warbringer
+	["ignore warcry cooldowns"] = function() return {mod("CooldownRecovery", "BASE", -8, { type = "SkillType", skillType = SkillType.Warcry })} end,
```

Comment:
This should use a `MAX` of 0 instead of `BASE` in case more warcries are added with different base cooldown values.